src/stable/lib/globals.ts

```diff
@@ -12,42 +12,51 @@ import { Action } from './state';
 
 declare global {
     // eslint-disable-next-line no-var
-    var _azleActions: Action[];
+    var _azleActions: Action[] | undefined;
     // eslint-disable-next-line no-var
     var _azleCanisterClassMeta: CanisterClassMeta | undefined;
     // eslint-disable-next-line no-var
-    var _azleCanisterMethodNames: { [key: string]: string };
+    var _azleCanisterMethodNames: { [key: string]: string } | undefined;
     // eslint-disable-next-line no-var
-    var _azleDispatch: (action: Action) => void;
+    var _azleDispatch: ((action: Action) => void) | undefined;
     // eslint-disable-next-line no-var
-    var _azleExperimental: boolean;
+    var _azleExperimental: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIcExperimental: AzleIcExperimental | undefined;
     // eslint-disable-next-line no-var
-    var _azleIcpReplicaWasmEnvironment: boolean;
+    var _azleIcpReplicaWasmEnvironment: boolean | undefined;
     // eslint-disable-next-line no-var
     var _azleIc: AzleIc | undefined;
     // eslint-disable-next-line no-var
-    var _azleInitCalled: boolean;
+    var _azleInitCalled: boolean | undefined;
     // eslint-disable-next-line no-var
-    var _azleNodejsWasmEnvironment: boolean;
+    var _azleNodejsWasmEnvironment: boolean | undefined;
```

Comment:
src/experimental/lib/stable_structures/stable_b_tree_map.ts not true in this file for this variable is subtly different than not undefined. Could you double check that?

Reply:
undefined is not true, so aren't we good?

Reply:
maybe... that what I wanted you to double check... but either way it's experimental